tools/src/schedule/archlinux.rs

```diff
@@ -133,12 +133,10 @@ pub async fn sync(sync: &PkgsSync) -> Result<Vec<PkgGroup>> {
     };
 
     let client = reqwest::Client::new();
-
     let mut bases: HashMap<_, PkgGroup> = HashMap::new();
     for arch in &sync.architectures {
         let db = mirror_to_url(source, &sync.suite, arch, &format!("{}.db", sync.suite))?;
-        let bytes = fetch_url_or_path(&client, &db)
-            .await?;
```

Comment:
Curious, is this the work of some code formatting plugin in your editor?

Reply:
I'm not familiar with rust code formatting opinions to answer if it's correct or incorrect, but the change itself felt like it was some autoformatter rather than you changing it around. IMO we can leave it out of the diff :)